src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,160 @@
+          activeClassName: 'mdc-list-item--activated',
+          exact: true,
+        }}
+        onClick={() => closeDrawer()}
+      >
+        {content}
+      </ListItem>
+    );
+  }
+
+  return (
+    <>
+      <TopAppBar>
+        <TopAppBarRow>
+          <TopAppBarSection alignStart>
+            {/*
+              TODO: the menu button is not very accessible like this:
+               A screen reader doesn't know that this button controls the drawer below.
+               The navigation items should be a sibling of this button.
+               But that is not easy to achieve right now with MDC.
+               Opened an issue: https://github.com/material-components/material-components-web/issues/4704
+            */}
+            {/*
+              TODO: switch to tag="button" later again because there currently is a bug with it.
+               Context: https://github.com/jamesmfriedman/rmwc/issues/459
+               */}
+            <TopAppBarNavigationIcon
+              role="button"
+              aria-expanded={open}
+              onClick={() => setOpen(!open)}
+            >
+              <div>
+                <Icon
+                  icon={{
+                    icon: icons.menu,
+                    size: 'medium',
+                    'aria-hidden': true,
+                  }}
+                />
+              </div>
+              <span className="sr-only">Menu</span>
+            </TopAppBarNavigationIcon>
+            <TopAppBarTitle
+              className={styles.title}
+              {...{ tag: Link, to: routes.Home }}
+            >
+              Ultimate Tic-Tac-Toe
+            </TopAppBarTitle>
+          </TopAppBarSection>
+          <TopAppBarSection alignEnd>
+            <TopAppBarActionItem
+              tag="a"
+              target="_blank"
+              rel="noopener noreferrer"
+              href={routes.GitHub}
+            >
+              <div>
+                <Icon
+                  icon={{
+                    icon: icons.github,
+                    size: 'medium',
+                    'aria-label': 'GitHub Icon',
+                  }}
+                />
+              </div>
+            </TopAppBarActionItem>
+          </TopAppBarSection>
+        </TopAppBarRow>
+      </TopAppBar>
+      <TopAppBarFixedAdjust />
+      <Drawer tag="nav" aria-label="Site" dismissible open={open}>
+        <DrawerContent>
+          <List>
+            {getNavigationItem('Play', icons.game, routes.Home)}
+            {getNavigationItem(
+              'Analysis',
+              icons.history,
+              routes.AnalysisOverview,
+            )}
+            {getNavigationItem(
+              'Analyse Last Game',
+              icons.replay,
+              routes.AnalysisLatest,
+            )}
+            <ListDivider />
+            {getNavigationItem(
+              'Report Bug',
+              icons.bug,
+              routes.GitHub + '/issues/new?template=bug_report.md',
```

Comment:
Should we add a route for Feature Requests aswell or should we replace this with a link to /issues? (so that you can also see the other issues and aren't linked directly to a GitHub login page)

Reply:
maybe some route like known issues